odo/backends/csv.py

```diff
@@ -366,8 +367,19 @@ def resource_glob(uri, **kwargs):
 @convert.register(chunks(pd.DataFrame), (chunks(CSV), chunks(Temp(CSV))),
                   cost=10.0)
 def convert_glob_of_csvs_to_chunks_of_dataframes(csvs, **kwargs):
```

Comment:
No, this is an internal function so it is fine. Also, the signature is implied because it is a `convert` dispatch